### ISSUE TITLE
No definition for fragmentation analysis

### ISSUE THREAD

**Issue by nascif** [adamtornhill/code-maat#11]
I found about it by accident (in an error message). I tried to infer what it does from the code but it would be nicer to have an example and proper documentation.

**Comment by adamtornhill** [comment]
Yes, I'd like to include a complete list of all analyses in the help text to start with. I think there are a few more I haven't documented. I'd also like to document them all here on github.
